buildtest/schemas/examples/script.schema.json/valid/examples.yml

```diff
@@ -234,6 +234,15 @@ buildspecs:
     skip: true
     run: hostname
 
+  summary_example:
+    type: script
+    executor: generic.local.bash
+    description: Short description limited to 80 characters
```

Comment:
Change this description to 

"The summary field can be a multi-line string and exceed 80 char"

Note this test is not meant to test description its for summary. There is some tests above for description if you look in file above